docusaurus/docs/reference/base/landUse.mdx

```diff
@@ -8,11 +8,35 @@ import generateResolverOptions from "@site/src/components/shared-libs/generateRe
 import yamlLoad from "@site/src/components/yamlLoad"
 import LandUseSchema from "!!raw-loader!@site/docs/_schema/base/landUse.yaml";
 
+import LandUseExample from "!!raw-loader!@site/docs/_examples/base/landUse-example.yaml";
+import LandUseExample2 from "!!raw-loader!@site/docs/_examples/base/landUse-example.yaml";
+
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Land Use
 
+Features in the land use theme come primarily from OpenStreetMap features containing the `landuse` tag.
+
```

Comment:
I wonder if we should adopt some kind of convention around OSM tags?

I often see them written as `landuse=*`...